src/EA.Weee.RequestHandlers.Tests.Unit/AatfReturn/GetReturnHandlerTests.cs

```diff
@@ -131,7 +131,7 @@ public async Task HandleAsync_GivenReturn_ObligatedValuesShouldBeRetrieved()
 
             var result = await handler.HandleAsync(new GetReturn(returnId));
 
-            A.CallTo(() => fetchObligatedWeeeDataAccess.FetchObligatedWeeeForReturn(returnId)).MustHaveHappened(Repeated.Exactly.Once);
+            A.CallTo(() => fetchObligatedWeeeDataAccess.FetchObligatedWeeeReceivedForReturn(returnId)).MustHaveHappened(Repeated.Exactly.Once);
```

Comment:
Add a similar test to check the re-used values are retrieved

Reply:
Reused check added, mapper test updated to also check for received and reused values